tests/bud.bats

```diff
@@ -1358,6 +1358,16 @@ function _test_http() {
   expect_output --substring "FROM alpine"
 }
 
+@test "bud with build arg ENV set in local environment" {
+   _prefetch alpine busybox
+  target=busybox-image
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} --build-arg foo=bar ${TESTSDIR}/bud/build-arg
+  expect_output --substring "bar"
+  export foo=hello-World
+  run_buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} --build-arg foo ${TESTSDIR}/bud/build-arg
+  expect_output --substring "hello-world"
+}
```

Comment:
IDK, but good to add this?
```
run_buildah bud --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/build-arg
expect_output --substring ""
```
It should just set foo to nothing.  Not sure if it's absolutely necessary, your call.

Reply:
Added test for this case as well :)